app/src/main/java/org/liberty/android/fantastischmemo/ui/MiscTabFragment.java

```diff
@@ -362,6 +366,13 @@ public void onClick(DialogInterface arg0, int arg1){
         if(v == aboutButton){
             aboutUtil.createAboutDialog();
         }
+        if(v == favouriteButton){
+            ((AnyMemo)getActivity()).getAllFavourites();
+            Intent myIntent = new Intent();
+            myIntent.setClass(getContext(), StudyActivity.class);
+            myIntent.putExtra(StudyActivity.EXTRA_DBPATH,favouritesDbPath );
```

Comment:
fix spacing in "myIntent.putExtra(StudyActivity.EXTRA_DBPATH,favouritesDbPath );"